### ISSUE TITLE
Tab completion with configuration for multiple vendors

### ISSUE THREAD

**Issue by krihal** [clicon/clixon-controller#40]
Environment with mixed vendors might result in different devices expecting different configuration.

Tab-completion will pick the configuration from the first device, this should result in an error stating that we have multiple vendors.

**Comment by olofhagsand** [comment]
@krihal please verify.
Maybe fixes #41 as well?

**Comment by krihal** [comment]
Definetly fixes #41, closed it.

Seeing some debug prints, except that it seems to work fine:

```
debian@khn-dev[/]# set devices device * conf
controller_cligen_treeref_wrap mountpoint-crpd1
controller_cligen_treeref_wrap mountpoint-crpd2
controller_cligen_treeref_wrap mountpoint-openconfig1
controller_cligen_treeref_wrap mountpoint-openconfig2
debian@khn-dev[/]# set devices device * config
```
